tasks/e2e/cypress/integration/01-tutorial/tutorial.spec.js

```diff
@@ -290,64 +303,64 @@ describe('The Redwood Tutorial - Golden path edition', () => {
     // {name: "test name", email: "foo@bar.com", message: "test message"}
   })
 
-  it('8. Saving Data', () => {
-    // navigate back out
-    cy.visit('http://localhost:8910/')
+  // it.skip('8. Saving Data', () => {
```

Comment:
These tests have not been flaky elsewhere, which leads me to believe something is happening in this PR that's causing true fails.

Are you just putting them on hold for now while you debug?

Reply:
I spoke a bit about this last week, I'm sure they've been flakey, the retries just made us not notice them:

There's a race condition because these tests modify the api side, but then whilst the api side is reloading, they hit the web side, and look for logs that are written by the api side.

Reply:
I added this check-mark, last night: `Determine how to match coverage for e2e tests that are now skipped.`

I'm not about reducing coverage.

Reply:
Roger that across the board. And 💯 makes sense.

I think I misunderstood where/how you were addressing the race condition.

Reply:
Ah totally, I mentioned that chokidar-http-server thing! But, I figured these needs to be trustable.